README.md

```diff
@@ -1,2 +1,59 @@
-# Portfolio
-Portfolio website that describes me and  what I do 
+# Portfolio:
+
+Portfolio website that describes me and what I do as software developer, designed using html and css.
+
+![screenshoot of the site](/img/toolbar_headline.PNG)
+It has got a toolbar that includes my logo and headline that talks about me.
+
+# Built with:
+
+- HTML
+- CSS
+
+# Getting started:
+
+To get a local copy up and running follow these simple example steps:
+
+- clone the repo using git clone or download and extract the file
+- Run the index.html file in browser (firefox)
+
+# Prerequisites:
+
+- computer
+- internet
+
+# Install
+
+- Install Web browser (firefox)
+- Install node latest version (from NPM)
+- Install code editor (visual studio)
+
+# Usage
+
+- clone the repo or download
+- open the files of the repo through code editor
+
+# Deployment
+
+- Run the index file on server
+
+# 👤 Author1:
+
+- [Github](https://github.com/ayebaishmo)
+- [Twitter](https://twitter.com/ishmo256)
+
+🤝 Contributing:
+
+- Contributions, issues, and feature requests are welcome!
+
+# Show your support
+
+Give a ⭐️ if you like this project!
+
+# Acknowledgments
+
+Thanks to figma microverse project- 1 provided a template of the site
+
+# 📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] [OPTIONAL] if you want to make your project look more professional, make sure when you include the license it is linked to the actual license adjusted for your project.